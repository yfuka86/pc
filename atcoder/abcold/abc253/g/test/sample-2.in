10 12 36
