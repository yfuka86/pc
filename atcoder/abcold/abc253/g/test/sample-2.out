1 10 9 8 7 4 3 2 5 6
